scripts/spellcheck.sh

```diff
@@ -107,7 +107,7 @@ checkFiles() {
 
     if [[ "$CHECK_GLOBAL_DOCS" = "true" ]]; then
         >&2 echo -n "global docs [md]... "
-        run_cspell "dev-docs/**/*.md" "components/automate-chef-io/content/docs/**/*.md" || RETVAL=1
+        run_cspell $(find . -name '*.md' | grep -v node_modules | grep -v "^\(\./cache\|\./components\|\./vendor\)") || RETVAL=1
```

Comment:
It seemed easier to use find and grep than deal with cspell's slow/broken globbing.

Reply:
My concern here is potentially hitting the command-line length limit. Though a quick search says it is 256KB, so I guess we're OK.

Reply:
There are not currently too many .md files in the project so this shouldn't cause any problems:

```
Petes-MBP:automate pete$ find . -name '*.md' | grep -v node_modules | grep -v "^\(\./cache\|\./components\|\./vendor\)" | wc -l
      53
```